feature-libs/cart/base/core/facade/active-cart.service.spec.ts

```diff
@@ -308,6 +332,28 @@ describe('ActiveCartService', () => {
     });
   });
 
+  describe('detectUserChange', () => {
+    it('should change loading flag to false if logged in with code flow', () => {
+      winRef.localStorage.setItem('oauthRedirectCodeFlow', 'true');
```

Comment:
Just nitpick :) Let's follow naming convention, so we should use: `oAuthRedirectCodeFlow` everywhere.

Reply:
Make sense, changed